pyvrp/Model.py

```diff
@@ -298,6 +299,9 @@ def solve(self, stop: StoppingCriterion, seed: int = 0) -> Result:
             for _ in range(pop_params.min_pop_size)
         ]
 
-        gen_args = (data, pm, rng, pop, ls, srex, init)
+        # We use SREX when the instance is a proper VRP; else OX for TSP.
+        crossover = srex if data.num_vehicles > 1 else ox
```

Comment:
`Model.solve` now automatically does the right thing. If one's building their own solve routine, I think it's reasonable to assume they also know how to select an appropriate crossover operator. The exception (in OX) and warning (in SREX) should help guide that choice.